src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -140,4 +145,18 @@ private function loadValues(string $file): array
 
         return (new Dotenv())->parse(file_get_contents($filePath));
     }
+
+    private function getDotenvFile(): string
+    {
+        $projectDir = is_file($projectDir = $this->projectDirectory) ? basename($projectDir) : $projectDir;
+
+        $composerFile = $projectDir.'/composer.json';
```

Comment:
I see the `dotenv_path` can be configured for Symfony Runtime in `APP_RUNTIME_OPTIONS` env var. We can look at it too, but I don't what is the priority.
https://github.com/symfony/symfony/blob/ac88f9a72bd0980ebff2bf4a24a11bdbc2aa6735/src/Symfony/Component/Runtime/SymfonyRuntime.php#L36-L39
https://github.com/symfony/symfony/blob/ac88f9a72bd0980ebff2bf4a24a11bdbc2aa6735/src/Symfony/Component/Runtime/Internal/autoload_runtime.template#L15-L16